### PR TITLE
ETSWORK-10254 - Bump dependencies to latest versions

### PR DESCRIPTION
**Pull request by coreyjv**
Bumping dependencies to latest versions to keep project in alignment and prevent significant drift. I had to add a symlink due to changes in Cypress. Additionally removed setting `aria-disabled` explicitly in `Radio` as that was fixed in Ionic `5.6.12`. Finally, a change in `react-hook-form` meant that I had to change how we provided the Smart Notification input type to the custom resolver. The `ref` no longer allows us to access the `data` attributes. I added an additional function to the `validate` in the `rules` to grab the element type as this seemed the most appropriate place to put it.

### CODE REVIEW

File: cypress/support/index.js [hunk 1]
```diff
@@ -17,7 +17,7 @@
 import './commands'
 
 // Ionic CSS setup
-import '../../ionic'
+import './ionic'
```
Comment (coreyjv) [inline cypress/support/index.js:20]: Cypress does not allow relative references outside of the "source directory". Per the error message I've added a symlink to the common configuration.

File: src/components/Radio.tsx [hunk 2]
```diff
@@ -40,8 +40,7 @@ export const Radio = ({ name, label, caption, color = 'medium', disabled, value
         {label}
         {caption && <p>{caption}</p>}
       </IonLabel>
-      {/* Added aria-disabled due to: https://github.com/ionic-team/ionic-framework/issues/23513 */}
-      <IonRadio color={color} name={name} disabled={disabled} value={value} aria-disabled={disabled} />
+      <IonRadio color={color} name={name} disabled={disabled} value={value} />
```
Comment (coreyjv) [inline src/components/Radio.tsx:43]: Ionic fixed this in `5.6.12` so we no longer need to explicitly specify `aria-disabled`

File: cypress/support/index.js (hunk 1 above)
Comment (TuckerWhitehouse) [reply cypress/support/index.js:20]: I wonder why they do this...

**Review by TuckerWhitehouse — APPROVED** [review]

**Review by louise-hayes — APPROVED** [review]

File: cypress/support/index.js (hunk 1 above)
Comment (coreyjv) [reply cypress/support/index.js:20]: I know it's certainly odd.

**Review by stevetothdesign — APPROVED** [review]

**Pull request merged**